repository/state/state.go

```diff
@@ -17,24 +17,35 @@
 package state
 
 import (
+	"bytes"
 	"encoding/binary"
 	"fmt"
 	"io"
 	"iter"
 	"sync"
 	"time"
 
+	"github.com/PlakarKorp/plakar/caching"
 	"github.com/PlakarKorp/plakar/objects"
 	"github.com/PlakarKorp/plakar/packfile"
+	"github.com/vmihailenco/msgpack/v5"
 )
 
 const VERSION = 100
 
+type EntryType uint8
+
+const (
+	ET_METADATA EntryType = iota
+	ET_LOCATIONS
+	ET_TIMESTAMP
+)
```

Comment:
micro nit: for values that are going to hit the disk I think it'd be better to use explicit values instead of iota; makes it easier not to break the on-disk format inadvertently when adding values.

Reply:
agreed, missed that